src/glyph/mod.rs

```diff
@@ -104,6 +104,28 @@ impl Glyph {
         }
     }
 
+    /// Returns boolean value indicating whether [`Glyph`] is defined with one or
+    /// more [`Component`].
+    pub fn has_component(&self) -> bool {
+        !self.components.is_empty()
+    }
+
+    /// Returns a usize that represents the number of [`Component`] defined on the Glyph.
+    pub fn component_number(&self) -> usize {
```

Comment:
I have a minor preference for:

```suggestion
    pub fn component_count(&self) -> usize {
```

Reply:
Ty!